tesseract_ros/tesseract_monitoring/include/tesseract_monitoring/environment_monitor.h

```diff
@@ -338,10 +345,10 @@ class EnvironmentMonitor
   ContinuousContactManagerPluginLoaderPtr continuous_manager_loader_;
 
   tesseract::Tesseract::Ptr tesseract_;
-  std::shared_mutex scene_update_mutex_;  /// mutex for stored scene
-  ros::Time last_update_time_;            /// Last time the state was updated
-  ros::Time last_robot_motion_time_;      /// Last time the robot has moved
-  bool enforce_next_state_update_;        /// flag to enforce immediate state update in onStateUpdate()
+  mutable std::shared_mutex scene_update_mutex_;  /// mutex for stored scene
```

Comment:
Why this change?

Reply:
Need to change it from a const function, because I need to lock it for read.